scripts/integration_tests.py

```diff
@@ -652,13 +653,18 @@ def test_vector_search_with_filter(primary, source_table):
 @pytest.fixture
 def external_index(request):
     cli_path = os.getenv("LANTERN_CLI_PATH")
+    use_ssl = os.getenv("EXTERNAL_INDEX_SECURE") == "1"
```

Comment:
Can this variable be called `USE_SSL` or whatever we decide is the uniform name of the variable?